server/src/test/java/org/eclipse/lsp/cobol/usecases/TestDAFGetEntityStatement.java

```diff
@@ -0,0 +1,96 @@
+          + "        PROGRAM-ID. test1. \r\n"
+          + "        DATA DIVISION. \r\n"
+          + "        WORKING-STORAGE SECTION. \r\n"
+          + "        01 {$*WS-AREA}. \r\n"
+          + "           03 {$*AREA-XW1}. \r\n"
+          + "             05 {$*DET001-XW1}. \r\n"
+          + "               07 FILLER               PIC X(5)    VALUE 'REMBD'. \r\n"
+          + "        PROCEDURE DIVISION. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} OWNER. \r\n"
+          + "            GET ENTITY 'TAKS' {$DET001-XW1} 'OWNER'. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} OWN. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} 'OWN'. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} DESIGNER. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} 'DESIGNER'. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} AVG. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} 'AVG'. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} ANALIST. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} 'ANALIST'. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} ANA. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} 'ANA'. \r\n"
+          + "            GET ENTITY 'TAK' 'DECENT-INKOOP' OWNER. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} DESCRIPTION 'VD'. \r\n"
+          + "            GET ENTITY DOM {$DET001-XW1} DESCRIPTION. \r\n"
+          + "            GET ENTITY 'DOM' {$DET001-XW1} DESCRIPTION. \r\n"
+      // Negative tests
+          + "            GET ENTITY {'TA'|2} {$DET001-XW1} OWNER. \r\n"
+          + "            GET ENTITY {'TAKSE'|2} {$DET001-XW1} 'OWNER'. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} {'OWNERT'|3}. \r\n"
+          + "            GET ENTITY 'TAK' {$DET001-XW1} DESCRIPTION {'VDW'|1}. \r\n"
+          + "            GET ENTITY {'DOMA'|4} {$DET001-XW1} DESCRIPTION. \r\n";
+
+  @Test
+  void test() {
+
+    Map<String, Diagnostic> diagnosticMap = new HashMap<>();
+    diagnosticMap.put(
+        "1",
+        new Diagnostic(
+            null,
+            "Exact length of tal must be 2 bytes",
+            DiagnosticSeverity.Error,
+            SourceInfoLevels.ERROR.getText()));
+    diagnosticMap.put(
+        "2",
+        new Diagnostic(
+            null,
+            "String length must be between 3 and 4",
+            DiagnosticSeverity.Error,
+            SourceInfoLevels.ERROR.getText()));
+    diagnosticMap.put(
+        "3",
+        new Diagnostic(
+            null,
+            "Only allowed value(s) OWNER,OWN,DESIGNER,AVG,ANALIST,ANA",
+            DiagnosticSeverity.Error,
+            SourceInfoLevels.ERROR.getText()));
+    diagnosticMap.put(
+        "4",
+        new Diagnostic(
+            null,
+            "Only allowed value(s) DOM",
+            DiagnosticSeverity.Error,
+            SourceInfoLevels.ERROR.getText()));
```

Comment:
Here and further the diagnostics definition may be simplified if there are fewer than 5 pairs using the `ImmutableMap.of()`

Reply:
Done.